package.json

```diff
@@ -53,102 +53,89 @@
     "serve:angular": "nx serve angular",
     "prepack": "yarn build:core && yarn build:all",
     "test": "nx run-many --target=test --all",
-    "postinstall": "node ./decorate-angular-cli.js && ngcc --properties es2015 browser module main && yarn build:core",
-    "ng": "nx"
+    "postinstall": "ngcc --properties es2020 browser module main"
   },
   "dependencies": {
-    "@angular/animations": "~13.2.0",
-    "@angular/common": "~13.2.0",
-    "@angular/compiler": "~13.2.0",
-    "@angular/core": "~13.2.0",
-    "@angular/forms": "~13.2.0",
-    "@angular/platform-browser": "~13.2.0",
-    "@angular/platform-browser-dynamic": "~13.2.0",
-    "@angular/router": "~13.2.0",
-    "@jscutlery/semver": "2.21.6",
-    "@ledgerhq/hw-transport": "^6.20.0",
-    "@ledgerhq/hw-transport-webhid": "^6.20.0",
-    "@ledgerhq/logs": "^6.10.0",
+    "@angular/animations": "~14.0.0",
+    "@angular/common": "~14.0.0",
+    "@angular/compiler": "~14.0.0",
+    "@angular/core": "~14.0.0",
+    "@angular/forms": "~14.0.0",
+    "@angular/platform-browser": "~14.0.0",
+    "@angular/platform-browser-dynamic": "~14.0.0",
+    "@angular/router": "~14.0.0",
+    "@jscutlery/semver": "^2.25.2",
+    "@ledgerhq/hw-transport": "^6.27.1",
+    "@ledgerhq/hw-transport-webhid": "^6.27.1",
     "@meteorwallet/sdk": "^0.0.9",
     "@nightlylabs/connect": "0.0.23",
-    "@nrwl/next": "13.8.5",
     "@walletconnect/qrcode-modal": "2.0.0-alpha.20",
     "@walletconnect/sign-client": "2.0.0-beta.101",
-    "big.js": "^6.1.1",
-    "bn.js": "^5.2.0",
-    "buffer": "^6.0.3",
+    "better-sqlite3": "^7.6.2",
+    "big.js": "^6.2.0",
+    "bn.js": "^5.2.1",
     "core-js": "^3.6.5",
-    "is-mobile": "^3.0.0",
-    "near-api-js": "0.44.2",
-    "next": "12.1.0",
-    "ngx-deploy-npm": "^3.1.7",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "regenerator-runtime": "^0.13.9",
-    "rxjs": "~7.4.0",
-    "tslib": "^2.0.0",
+    "is-mobile": "^3.1.1",
+    "jest-preset-angular": "^12.2.0",
+    "near-api-js": "^0.45.1",
+    "next": "12.1.6",
+    "ngx-deploy-npm": "^4.1.1",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "regenerator-runtime": "0.13.7",
+    "rxjs": "^7.5.5",
+    "tslib": "^2.3.0",
     "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~13.2.0",
-    "@angular-eslint/eslint-plugin": "~13.0.1",
-    "@angular-eslint/eslint-plugin-template": "~13.0.1",
-    "@angular-eslint/template-parser": "~13.0.1",
-    "@angular/cli": "~13.2.0",
-    "@angular/compiler-cli": "~13.2.0",
-    "@angular/language-service": "~13.2.0",
-    "@babel/preset-react": "^7.16.7",
-    "@nrwl/angular": "^13.8.5",
-    "@nrwl/cli": "13.8.5",
-    "@nrwl/cypress": "13.8.5",
-    "@nrwl/eslint-plugin-nx": "13.8.5",
-    "@nrwl/jest": "13.8.5",
-    "@nrwl/js": "13.8.5",
-    "@nrwl/linter": "13.8.5",
-    "@nrwl/next": "13.8.5",
+    "@angular-devkit/build-angular": "~14.0.0",
```

Comment:
There are some devDependencies missing:
You need to add them with `yarn add --dev` for example the first one is something like this:

`yarn add --dev @babel/preset-react@^7.16.7`

This way we install the versions we had before.

```
    "@babel/preset-react": "^7.16.7",
    "@nrwl/tao": "13.8.5",
    "@testing-library/react-hooks": "7.0.2",
    "@types/regenerator-runtime": "^0.13.1",
    "@types/w3c-web-hid": "^1.0.2",
    "@types/w3c-web-usb": "^1.0.5",
    "jest-preset-angular": "11.1.1",
    "gh-pages": "^3.2.3",
    "semantic-release": "^19.0.2",
    "@semantic-release/commit-analyzer": "^9.0.2",
    "@semantic-release/git": "^10.0.1",
    "@semantic-release/npm": "^9.0.0",
    "@testing-library/jest-dom": "^5.16.1",
    
   ```